service-workers/service-worker/referer-toplevel-script-fetch.https.html

```diff
@@ -1,10 +1,12 @@
 <!DOCTYPE html>
 <title>Service Worker: check referer of top-level script fetch</title>
+<meta name="timeout" content="long">
```

Comment:
@KyleJu the test failed on Taskcluster because it was too slow (it *almost* timed out). There's a heuristic in the stability check to prevent near timeouts from landing. Giving the test a long timeout should solve the problem.